package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "convergame-engine",
+  "version": "0.0.1",
+  "license": "GLP3",
```

Comment:
Lesser GPL3.

Not sure on the syntax to represent this in `package.json`. I'd assume it is either LGPL3 or LGPLv3.